packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantLogo/_tests/RestaurantLogo.test.js

```diff
@@ -0,0 +1,57 @@
+import { mount, config } from '@vue/test-utils';
+import RestaurantLogo from '../RestaurantLogo.vue';
+
+// Allows for the testing of CSS module classes
+config.mocks.$style = new Proxy({}, {
+    get (_, name) {
+        return name;
+    }
+});
```

Comment:
at the very least I'd write this once for the component repository and import it where needed. Rather than keep rewriting it